package.json

```diff
@@ -69,8 +69,7 @@
   },
   "options": {
     "coverage": "--reporter=text-lcov",
-    "mocha": "--require ./babel.test.js --timeout 20000 --recursive",
-    "standard": "\"**/*.js\" \"**/*.html\" \"**/*.md\""
```

Comment:
lint did not work.

Reply:
Yep it's working after this change. not sure why actually... the format seemed fine.